libevm/VMOpt.cpp

```diff
@@ -63,14 +63,16 @@ void VM::copyCode(int _extraBytes)
 
 void VM::optimize()
 {
+	copyCode(33);
```

Comment:
I changed it here to the way it was before recent modifications -- reserving `codeSize + 33` bytes, copying all the code.
I'm not sure it's correct, and what was the purpose of reserving extra `pc - nBytes + 33` bytes (maybe it was optimization to reserve less in case the for loop hits the break on line 120?)